app/models/preference.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Preference model definition
+class Preference < ApplicationRecord
+  belongs_to :item
+  validates :multiplier, presence: true, numericality: { greater_than_or_equal_to: 1 }
```

Comment:
We may want to make this a fractional multiplier in the future, but this works for now.